app/src/main/java/com/jerboa/DefaultInstances.kt

```diff
@@ -0,0 +1,60 @@
+package com.jerboa
+
+val DEFAULT_LEMMY_INSTANCES = arrayOf(
+    "lemmy.world", // 13856 monthly users
+    "lemmy.ml", // 4206 monthly users
+    "beehaw.org", // 3730 monthly users
+    "feddit.de", // 2300 monthly users
+    "sh.itjust.works", // 2215 monthly users
+    "www.hexbear.net", // 1603 monthly users
+    "lemmynsfw.com", // 1122 monthly users
```

Comment:
How do other apps handle this then which have NSFW? I would argue that Jerboa supports NSFW as much as a browser does. Which is to say, is obviously not its target.

But the only real gripe is that I would have, is that the only way to do so is, is by keeping a list of nfsw instances. Defeating mostly the point of this pr, which is to prevent keeping a manual list.

Reply:
Jerboa supports NSFW as much as the browser does, but a browser also isn't going to have NSFW in its default bookmark list, which this is comparable to. It is unfortunate, but at least this list should be smaller, and we only need to evaluate instances in the list.